code/game/antagonist/antagonist_objectives.dm

```diff
@@ -41,25 +41,24 @@
 
 	if(!mind)
 		return
+
 	if(!is_special_character(mind))
-		to_chat(src, "<span class='warning'>While you may perhaps have goals, this verb's meant to only be visible \
-		to antagonists.  Please make a bug report!</span>")
+		to_chat(src, SPAN_WARNING("While you may perhaps have goals, this verb's meant to only be visible \
+		to antagonists. Please make a bug report!"))
 		return
 
-	var/datum/goal/ambition/goal = SSgoals.ambitions[mind]
-	var/new_goal = sanitize(input(src, "Write a short sentence of what your character hopes to accomplish \
-	today as an antagonist.  Remember that this is purely optional.  It will be shown at the end of the \
-	round for everybody else.", "Antagonist Goal", (goal ? html_decode(goal.description) : "")) as null|message)
+	var/datum/goal/ambition/goals = SSgoals.ambitions[mind]
+	var/new_goal = sanitize(input(src, "Write down what you want to achieve in this round as an antagonist \
+	- your goals. They will be visible to all players after the end of the round. \
+	remember you cannot rewrite them - only add new lines.", "Antagonist Goal") as null|message)
```

Comment:
It seems terrible that you can't rewrite them. Prepare for an eternity of "Can you modify my ambitions" ahelps.

Reply:
You can just add new one with info like "I decided to not do X".

Reply:
But what if you just made an error? I don't see how this is better than the current, more flexible setup.

Reply:
Editing ambitions partially will ruin diary effect, means you can rewrite your entire history.

Reply:
Current one is just a static text like "I want to do X" and all. New one allows to track special roles history and know how people change their tastes goals and exactly when.

Reply:
I'm with Afterthought, not letting them be freely editable is a downgrade from current function. You could achieve the same 'diary' thing by letting them add more ambitions to their goals list and optionally letting them specify a timestamp.

Overall not a fan of this PR as a mandatory thing.